### ISSUE TITLE
Investigate possible data distribution shifts between Phase I and Phase II

### ISSUE THREAD

**Issue by kvantricht** [WorldCereal/worldcereal-classification#116]
Based on a well-known region (e.g. Flanders), we query existing public geoparquet for samples, make a subset of **~2500** points.
Make use of scripts/extractions/point_extractions/point_extractions.py to extract Phase II preprocessed time series.

- [x] Save subset of Phase I parquet file
- [x] Generate identical Phase II parquet file
- [ ] Sanity checks on different input channels
- [ ] Make identical CAL/VAL/TEST set for Phase I and Phase II
- [ ] Already finetuned Presto: compute Phase I and Phase II embeddings
- [ ] Train CatBoost on Phase I CAL/VAL
- [ ] Compare trained CatBoost on Phase I TEST vs Phase II TEST
- [ ] Train CatBoost on Phase II CAL/VAL
- [ ] Deploy model and run inference for Flemish AOI and compare to Phase I CatBoost on this AOI

**Comment by kvantricht** [comment]
![Image](https://github.com/user-attachments/assets/c7fac99d-313e-482f-843f-8d162d3568dc)

**Comment by kvantricht** [comment]
More points and including density coloring: 



![Image](https://github.com/user-attachments/assets/c8677c83-b15a-4bb5-a5c0-d60aab24a0dc)

for now I'd say only Sentinel-1 is a bit suspicious. It's confirmed that we are not comparing exactly the same pixels, so scattering is definitely a possibility. Sentinel-1 though has a consistent bias. That means that without training on openEO-specific Sentinel-1 (Orfeo) is likely gonna lead to deteriorated model performance.

**Comment by kvantricht** [comment]
- When training CatBoost on original (non-Presto) inputs for Phase I vs Phase II, results are close to identical. So the same "amount of predictive signal" is present in both.

**Comment by kvantricht** [comment]
Main issues were related fixes in #111. Although the S1 distribution shift likely contributes to prediction deterioration, training on Phase II data in the future should fix this. Closing as nothing more to be done for now.
